tracim/tracim/templates/home.mak

```diff
@@ -90,11 +90,13 @@
 
     ## NOT READ
     <div class="" id="unread-content-panel">
+
         <div class="panel panel-success">
             <div class="panel-body">
                 % if fake_api.last_unread.nb <= 0:
                     ${P.EMPTY_CONTENT(_('No new content.'))}
                 % else:
+                    <a href="/tools/read/all" class="btn btn-default"> Mark everything as read </a>
```

Comment:
Move the endpoint to `/content/unread/mark_read`.

Endpoints implementing the same behavior for workspaces will be: /workspaces/4/mark_read